generators/ci-cd/index.js

```diff
@@ -81,17 +81,18 @@ module.exports = class extends BaseGenerator {
             },
             getConfig() {
                 this.jhipsterVersion = packagejs.version;
-                this.baseName = this.config.get('baseName');
-                this.applicationType = this.config.get('applicationType');
-                this.skipClient = this.config.get('skipClient');
-                this.clientPackageManager = this.config.get('clientPackageManager');
-                this.buildTool = this.config.get('buildTool');
-                this.herokuAppName = this.config.get('herokuAppName');
+                const configuration = this.getAllJhipsterConfig(this, true);
```

Comment:
I've seen this code somewhere, but I don't know why this work compared to what we already have!
What's the difference between these two approaches? thanks @sendilkumarn :)

Reply:
getAllJhipsterConfig is a workaround for https://github.com/jhipster/generator-jhipster/issues/10663.
It gets all configs from every namespace.

Reply:
I think I get it, thanks :)